src/page.ts

```diff
@@ -0,0 +1,119 @@
+import { IFilterable } from './ifilterable';
+import { IReport } from './report';
+import { Visual } from './visual';
+import * as models from 'powerbi-models';
+
+export interface IPage {
+    report: IReport;
+    name: string;
+    displayName: string;
+    setActive(): void;
+}
+
+export class Page implements IPage, IFilterable {
```

Comment:
Given my other comment, I think I was inconsistent here.   If we go with minimal interface I should remove `displayName` and `setActive` from `IPage` but if we go with class interface I would add `IFilterable`. I'm not sure which is better at the moment.